pkgs/top-level/all-packages.nix

```diff
@@ -4725,10 +4725,6 @@ in
 
   flamerobin = callPackage ../applications/misc/flamerobin { };
 
-  flashtool = pkgsi686Linux.callPackage ../development/mobile/flashtool {
-    inherit (androidenv.androidPkgs_9_0) platform-tools;
-  };
-
```

Comment:
Why is `flashtool` removed? Can't find a reason mentioned in the PR or commit

Reply:
@SuperSandro2000 suggested: https://github.com/NixOS/nixpkgs/pull/126894#discussion_r651379506

Reply:
Because the download is down and the site looks very fishy and very similar to the ones used to distribute content illegally.

Reply:
Fair enough. But the reason for this should be noted in the commit (and PR description) imho.